### ISSUE TITLE
A detail about data processing

### ISSUE THREAD

**Issue by ShuaibZyx** [CharlieCrosley/Polygen-Pytorch#3]
May I ask why “vertices = vertices[:, [2, 0, 1]]” is used during data processing？Thank you！

**Comment by CharlieCrosley** [comment]
This is just to change the order that the x,y,z coordinates appear in the list. This just rearranges the vertices so that the z axis is vertical

**Comment by ShuaibZyx** [comment]
If it is for the z-axis, why is the following code used in the following processing:
# Re-order vertex coordinates as (z, y, x).
 vertices_permuted = torch.stack(
        [vertices[:, 2], vertices[:, 1], vertices[:, 0]], axis=-1
 )
 This code is in the make vertex method in data utils.Thank you.

**Comment by CharlieCrosley** [comment]
The original Polygen paper says that the vertices are ordered by z then y then x, in ascending order. This helps to simplify the problem as the same mesh can have its vertices ordered differently.

**Comment by ShuaibZyx** [comment]
But what does code "torch.stack([vertices[:, 2], vertices[:, 1], vertices[:, 0]], axis=-1)" do is change the dimension from XYZ to ZYX?The function of the code "vertices = vertices[:, [2, 0, 1]]" is again to change the dimension from XYZ to ZXY. The result of these two pieces of code is to change the original coordinate XYZ first to ZXY, then to YXZ. Where is my understanding wrong, please correct. Thank you.

**Comment by CharlieCrosley** [comment]
Most of the code you are referring to is taken from the original implementation and just converted to PyTorch where necessary, so im not 100% sure on the intention behind it. However, I believe the line "vertices = vertices[:, [2, 0, 1]]" is swapping the axis around so that the object is oriented correctly, although it appears to result in the order ZXY, we say it is XYZ. This means that "torch.stack([vertices[:, 2], vertices[:, 1], vertices[:, 0]], axis=-1)" will actually result in the order ZYX. You can verify this by changing the order of the transpose and displaying the mesh to see if the orientation changes.

**Comment by ShuaibZyx** [comment]
Ok, I see. I will try to verrify this issue. Thank you for your reply, it helps me a lot. Thank you！
